docs/source/eFeatures.rst

```diff
@@ -150,20 +150,20 @@ time from stimulus start to last spike
     else:
         time_to_last_spike = 0
 
-`Python efeature`_ : Spikecount
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+`Python efeature`_ : spike_count
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 number of spikes in the trace, including outside of stimulus interval
 
 - **Required features**: LibV1:peak_indices
 - **Units**: constant
 - **Pseudocode**: ::
 
-    Spikecount = len(peak_indices)
+    spike_count = len(peak_indices)
 
 **Note**: In the future this feature will be called "spike_count".
```

Comment:
Could you please tell in the note that this feature replaces Spikecount, that Spikecount is deprecated, but can still be used for the moment?

Reply:
Ah yes, I also noticed I forgot to update this one. Done in the last commit. :+1: